packages/salesforcedx-vscode-core/package.json

```diff
@@ -908,11 +909,11 @@
     "jsonValidation": [
       {
         "fileMatch": "sfdx-project.json",
-        "url": "https://forcedotcom.github.io/schemas/sfdx-project.json/sfdx-project.schema.json"
+        "url": "./node_modules/@salesforce/schemas/sfdx-project.schema.json"
```

Comment:
I just want to confirm that this will always be the path right? i.e. does vscode currently or will do any module hoisting or linking (like yarn2) in the future? If so these paths would break.

Reply:
@amphro It doesn't appear that vscode does anything like that right now, it seems to just basically unzip the .vsix as-is. I have seen other extensions that reference files in ./node_modules directly programmatically; I thought I had seen this way in package.json, too, which is why I figured this would work (which it does).
I suppose vscode could change something in the future, but also, as far as I know, there is no programatic api to dynamically add schemas to the default json langserver; you have to do it through this package.json, which has to point to urls or relative file paths.